site/app/models/Gradeable.php

```diff
@@ -1168,6 +1169,10 @@ public function getComponentsGradedBy($grader_id) {
         return $return;
     }
 
+    public function getRegradeStatus(){
+        return $this->regrade_status;
+    }
+
```

Comment:
you don't need this function (though please add it to the top of the file in that big method comment block so that PHPStorm and IntelliJ recognize that it exists). The magic `__call()` function in the AbstractModel maps `getRegradeStatus()` to return `$this->regrade_status` automatically.